assets/src/js/admin/paypal-commerce/index.js

```diff
@@ -43,6 +44,7 @@ window.addEventListener( 'DOMContentLoaded', function() {
 						<ul>${ pciWarnings }</ul>
 					</div>
 				`.trim(),
+				cancelBtnTitle: Give.fn.getGlobalVar( 'confirm' ),
```

Comment:
@JasonTheAdams This code is to update modal button text and color: https://givewp.slack.com/archives/C0FAGC83C/p1598550190229100